ubtue_Sciendo.js

```diff
@@ -1,7 +1,7 @@
 {
 	"translatorID": "bf64f8a7-89b4-4a79-ae80-70630e428f35",
 	"label": "Sciendo",
-	"creator": "Madeesh Kannan",
+	"creator": "Madeesh Kannan", "Timotheus Kim",
```

Comment:
Are you sure this syntax works? seems to be invalid JSON to me.

Reply:
Indeed it is invalid. It should be "Madeesh Kannan and Timotheus Kim"